sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileScan.scala

```diff
@@ -37,22 +37,6 @@ abstract class FileScan(
     false
   }
 
-  /**
-   * Returns whether this format supports the given [[DataType]] in write path.
-   * By default all data types are supported.
-   */
-  def supportsDataType(dataType: DataType): Boolean = true
-
-  /**
-   * The string that represents the format that this data source provider uses. This is
-   * overridden by children to provide a nice alias for the data source. For example:
-   *
-   * {{{
-   *   override def formatName(): String = "ORC"
-   * }}}
-   */
-  def formatName: String
-
```

Comment:
@gengliangwang . This looks irrelevant to this PR. And, I cannot see the explanation about moving `formatName`. For me, `CSVScan` naturally has `def formatName`. Why do we need to move `formatName`?

Reply:
The `formatName` was added for showing the Exception message.
https://github.com/apache/spark/pull/23714#pullrequestreview-203520602

Reply:
Thanks. Yes. It seems to be moved together correctly.